scripts/core/menu/views/superdesk-view.html

```diff
@@ -14,28 +14,23 @@
 
 <div class="loading" ng-if="loading"></div>
 
-<div id="main-container" ng-class="superdesk.flags" ng-if="!popup && superdesk.flags.workspace === true"
-    ngf-drop="superdesk.openUpload($files)"
-    ngf-model-options="{updateOn: 'drop'}"
```

Comment:
@petrjasek any idea why `ngf-drop` stuff was here in the first place? It looks like it's trying to make the entire app as a drop target which doesn't make sense. I've removed this and uploading still works when adding media items to editor3/editor2, via "+" button in authoring as well as uploading avatars. Any ideas on what removing this might break?

Reply:
I think this is for uploading pictures via drag and drop

Reply:
I mean to create new items not adding to existing one

Reply:
I tried it again and I found how it works. I was trying to drop images in the top bar previously so it seemed not to work.